### PR TITLE
ECL IDE Find Repository "Refresh" duplicates attributes

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-742

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/RepositoryTreeNode.cpp [hunk 1]
```diff
@@ -535,9 +535,21 @@ CModuleFilterNode::CModuleFilterNode(IRepositorySlot *owner, IModule * mod) : CM
 {
 }
 
+bool CModuleFilterNode::FindAttribute(IAttribute *attr)
```
Comment (GordonSmith) [inline wlib/RepositoryTreeNode.cpp]: Rename to "HasAttribute"

**Comment by richardkchapman** [comment]
@dehilsterlexis

**Comment by dehilsterlexis** [comment]
@GordonSmith back to you

**Pull request merged**